src/routes/qumlBulkRoutes.js

```diff
@@ -0,0 +1,11 @@
+const qumlBulkUpload = require("../service/qumlBulkService");
+const BASE_URL = '/api/question/v1/';
+
+module.exports = function (app) {
+    app.route(BASE_URL + '/bulkUpload')
```

Comment:
`BASE_URL` is ending with the `/` and the route endpoint is starting with a `/` . we will ending up accessing the api like `question/v1//bulkUpload`. please fix.